tests/runtest.cmd

```diff
@@ -375,7 +380,7 @@ goto :eof
 
         set __ExtraTestRunArgs=
         if "%__Mode%"=="readytorun" (
-            set __Extension=ni.exe
+            set __Extension=exe
```

Comment:
There are more references to `ni.exe` in the test tree.

* The comments in the CMD files in individual tests will be stale after this change.
* There is a reference to ni.exe in build-and-run-test.cmd that appears to be dead code (?)

Reply:
Also, extension being exe is the default, set 10 lines above, so this is line is now a no-op and can be deleted.